src/frontends/common_translators/src/utils.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils.hpp"
+
+#include "openvino/frontend/complex_type_mark.hpp"
+#include "openvino/frontend/exception.hpp"
+
+namespace ov {
+namespace frontend {
+namespace common_translators {
+
+void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs, bool allow_complex) {
+    auto num_inputs = context.get_input_size();
+    FRONT_END_OP_CONVERSION_CHECK(num_inputs >= min_inputs,
```

Comment:
This behavior is different from pytorch FE behavior

Reply:
Aligned with PT FE, thanks